File: 백준/단계 별로 풀어보기/입출력과_사칙연산/10430_나머지.py
import sys
input = sys.stdin.readline
IP = lambda: map(int, sys.stdin.readline().split())

def solve():
    a, b, c = IP()
    fir = (a + b) % c
    sec = (a * b) % c
    print(fir, fir, sec, sec, sep = '\n')
    
if "__main__" == __name__: solve()

File: baekjoon/단계 별로 풀어보기/입출력과 사칙연산/10430_나머지.cpp
#include <stdio.h>
using namespace std;

int main()
{
	int a, b, c;
	scanf("%d %d %d", &a, &b, &c);
	int fir = (a + b) % c, sec = (a * b) % c;
	printf("%d\n%d\n%d\n%d\n", fir, fir, sec, sec);
}